impress-org/givewp
# Refactor: update price option settings control

Resolves: [GIVE-625]

## Description
https://app.zeplin.io/project/654141b179b50d20560405c8/screen/6616a7049fb6e9766e98a3e5

This PR aims to improve the priceOption setting controls from within the amount blocks inspector controls. 

Changes:
- Remove DonationLevels dropdown & move the amount options into DonationOptions dropdown. The conditional for which priceOption should show was updated to a ternary conditional.
- Replace the SelectControl WP component with a custom control using the proposed new design.

## Affects
Donation Amount Controls

## Visuals

https://github.com/impress-org/givewp/assets/75056371/db004dac-d451-49d9-b1b4-e49288fc4ac3

## Testing Instructions
- Toggle the new control
- Verify the correct donation amount input is displayed when the corresponding setting is chosen
- Verify the build preview updates with the correct donation amount preview
## Pre-review Checklist




[GIVE-625]: https://stellarwp.atlassian.net/browse/GIVE-625?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
